src/main/kotlin/app/revanced/patches/youtube/layout/theme/resource/ThemeResourcePatch.kt

```diff
@@ -18,16 +19,13 @@ import org.w3c.dom.Element
 @DependsOn([SettingsPatch::class, ResourceMappingPatch::class])
 class ThemeResourcePatch : ResourcePatch {
     override fun execute(context: ResourceContext): PatchResult {
-        SettingsPatch.PreferenceScreen.LAYOUT.addPreferences(
+        SeekbarColorResourcePatch.addSeekbarPreferences(
```

Comment:
`ThemeResourcePatch` does not depend on `SeekbarColorResourcePatch`. For that reason, `close` will never be called, unless another patch depends on it and is included. Add a dependency to `SeekbarColorResourcePatch` for patch `ThemeResourcePatch`.